sidhariyal/star_names.fab

```diff
@@ -0,0 +1,1181 @@
+# star names by constellation
+# Andromeda (And)
+   677|_("உத்திரட்டாதி 2/முரசு") 1,2,5,6,11,12,37,38
+   677|_("Sirrah") 23,37

```

Comment:
Overall, names not used in Tamil tradition should be deleted. To just show "Western" names translated to Tamil, translate them in Transifex.

Reply:
Noted with thanks, I'll incorporate it.